src/ctapipe/calib/camera/calibrator.py

```diff
@@ -31,7 +31,10 @@ def _get_invalid_pixels(n_pixels, pixel_status, selected_gain_channel):
     )
     for mask in masks:
         if mask is not None:
-            broken_pixels |= mask[selected_gain_channel, index]
+            if selected_gain_channel is not None:
+                broken_pixels |= mask[selected_gain_channel, index]
+            else:
+                broken_pixels |= mask
```

Comment:
You also need to change the shape of `broken_pixels` to be `(n_channels, n_pixels)`, since pixels might be broken only in one channel (e.g. LST channels of the same pixel are on different DAQ hardware boards)

Reply:
Should the `InvalidPixelHandler` then also be able to handle multiple gain channels?